### PR TITLE
US 64 Filter customers ınfo on account page

### PR DESCRIPTION
**Pull request by NurcanKavlak**
US64 is done .It failed .There is a bug

### CODE REVIEW

**Review by ValTsoy — APPROVED** [review]
I'm not good at coding and I Approved

**Review by sinacopur — COMMENTED** [review]
Good Job!

File: src/test/java/com/vytrack/tests/US64_FilterCustomersInfoOnAccountPage.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+            Assert.assertEquals(actualModuleTexts, expectedModuleTexts);
+
+        }
+
+        //testing modules for stores manager
+        @Test
+        public void filteringCustomersInfoForStoreManager() {
+            //Step1.login as Store Manager
+            VytrackUtils.loginAsStoreManger();
+            WebElement customerModule = Driver.getDriver().findElement(By.xpath("(//span[@class='title title-level-1'])[3]"));
```
Comment (Lilya04) [inline src/test/java/com/vytrack/tests/US64_FilterCustomersInfoOnAccountPage.java:114]: I would find element using name not index.
For example "//span[normalize-space()='Customers' and contains(@class='title title-level-1')]"

**Review by Lilya04 — COMMENTED** [review]
Looks good

File: src/test/java/com/vytrack/tests/US64_FilterCustomersInfoOnAccountPage.java (hunk 1 above)
Comment (ValTsoy) [reply src/test/java/com/vytrack/tests/US64_FilterCustomersInfoOnAccountPage.java:114]: Lilya I'm not sure can you please ask David Please?

Comment (ValTsoy) [reply src/test/java/com/vytrack/tests/US64_FilterCustomersInfoOnAccountPage.java:114]: Lilya Can you please ask someone from like Adam?

Comment (Lilya04) [reply src/test/java/com/vytrack/tests/US64_FilterCustomersInfoOnAccountPage.java:114]: Actually I have no question. It was just recommendation.

**Review by ValTsoy — COMMENTED** [review]
Ok

**Review by ValTsoy — COMMENTED** [review]
ok

**Pull request merged**